docs/apps/guacamole.md

```diff
@@ -1 +1,64 @@
-Placeholder page
+# This guide will help you follow Gilbn's tutorial to protect your Guacamole install with F2B.
+
+Since DockSTARTer uses Oznu's image for Guacamole, it only generates logs inside the container itself. Following these steps will allow you to get the Guacamole container to generate a log file in `~/.config/appdata/guacamole` which you can then mount to the LetsEncrypt container so F2B can monitor it and ban malicious IPs.
+
+You can find Gilbn's tutorial [here](https://technicalramblings.com/blog/remotely-accessing-the-unraid-gui-with-guacamole-and-vnc-web-browser/). You will need to follow it after completing the following steps:
+
+1. Create a `logback.xml` file inside `~/.config/appdata/guacamole/guacamole`
+    1. `touch ~/config/appdata/guacamole/guacamole`
+  <br> or
```

Comment:
`<br>` is not markdown. can you just use an extra blank line?

Reply:
Fixed.